index.js

```diff
@@ -378,10 +383,12 @@
         }
 
         let el = document.createElement('button');
-        el.innerText = 'Save in "Smart Baskets"';
+        const existsInStorage = cartExistsInStorage(getCurrentCart().id);
+        el.innerText = `${existsInStorage ? 'Update cart' : 'Save cart'} in "Smart Baskets"`;
         el.className = 'link-button smart-basket-save-button';
         el.addEventListener('click', () => {
             storeCurrentCart();
+            window.alert(`Cart "${getCurrentCart().name}" ${existsInStorage ? 'has been updated' : 'is now stored'} in "Smart Baskets"`)
```

Comment:
`window.alert` sucks, especially now that adding decent notification popups is a breeze (thanks npm)
Please consider using https://www.npmjs.com/package/notyf instead. 
LMK If you're not up to it, I'm happy to make the change